streams/src/test/java/org/apache/kafka/streams/processor/internals/ProcessorTopologyTest.java

```diff
@@ -280,61 +280,6 @@ public void testDrivingSimpleTopology() {
         assertTrue(outputTopic1.isEmpty());
     }
 
-
-    @Test
-    public void testDrivingMultiplexingTopology() {
```

Comment:
Just wondering, why do we remove this test (and the below)?

Reply:
It's using the "old API" topology provided by `createMultiplexingTopology` (and below `createMultiplexByNameTopology`), ie, `MultiplexByNameProcessor` and `MultiplexingProcessor`.

I did double check and the tested multiplexing case is tested using the new API via `createMultiProcessorTimestampTopology` that is using the `FanOutTimestampProcessor` that is the "new" equivalent of the old (and removed) `MultiplexByNameProcessor` processor (for the multiplex by index we don't need a new test as we removed routing by index on purpose and it's not part of the new API any longer.)